### PR TITLE
[BXMSPROD-1845] elasticsearch removed from appformer

### PR DESCRIPTION
**Pull request by Ginxo**
https://issues.redhat.com/browse/BXMSPROD-1845

Related PRs:

* https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/2097
* https://github.com/kiegroup/appformer/pull/1312
* https://github.com/kiegroup/kie-wb-common/pull/3781
* https://github.com/kiegroup/drools-wb/pull/1558
* https://github.com/kiegroup/jbpm-wb/pull/1572
* https://github.com/kiegroup/optaplanner-wb/pull/413
* https://github.com/kiegroup/kie-wb-distributions/pull/1196

### CODE REVIEW

**Comment by Ginxo** [comment]
Jenkins retest this

**Comment by Ginxo** [comment]
Jenkins run fdb

**Comment by Ginxo** [comment]
Jenkins run cdb

**Comment by Ginxo** [comment]
Jenkins run fdb

**Comment by Ginxo** [comment]
Jenkins retest this

**Comment by Ginxo** [comment]
Jenkins run fdb

**Comment by Ginxo** [comment]
FDB check -> https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/main/job/fdb/job/appformer-main.fdb/157/console

**Comment by Ginxo** [comment]
Jenkins run fdb

**Review by mareknovotny — APPROVED** [review]
great, so we could upgrade later lucene to the latest version too if used just for indexing purposes outside ES

File: pom.xml [hunk 1]
```diff
@@ -1075,15 +1045,6 @@
...
 
-      <!-- Required to run the ElasticSearch server when performing the unit test executions.
-          The ELS data provider requires groovy scripts enabled on the server.-->
-      <dependency>
-        <groupId>org.codehaus.groovy</groupId>
-        <artifactId>groovy-all</artifactId>
-        <classifier>indy</classifier>
-        <version>${version.org.codehaus.groovy}</version>
-      </dependency>
-
```
Comment (mareknovotny) [inline pom.xml:1086]: it seems this is needed in kie-soup?
```
[2022-10-07T09:53:01.927Z] [ERROR] 'dependencies.dependency.version' for org.codehaus.groovy:groovy-all:jar:indy must be a valid version but is '${version.org.codehaus.groovy}'. @ org.kie.soup:kie-soup-parent:7.74.0-SNAPSHOT, /home/jenkins/workspace/KIE/main/pullrequest/drools-wb-main.pr/bc/kiegroup_kie_soup/pom.xml, line 498, column 18
[2022-10-07T09:53:01.927Z] 
```

**Comment by Ginxo** [comment]
Jenkins retest this

**Comment by Ginxo** [comment]
FDB Unstable due to:

- [droolsjbpm-integration] LoadBalancerClientTest.testDefaultLoadBalancerNoServersAvailable:279 » NoEndpointFound
- [kie-wb-common] ModuleDataModelFactAnnotationsTest.annotationsWithMemberOfTypeClass:140 expected:<1> but was:<0>
- [droolsjbpm-integration] 3 additional integration tests

They don't seem related with the change itself

**Review by adrielparedes — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (adrielparedes) [reply pom.xml:1086]: that could be part of Dashbuilder @jesuino

**Review by lampajr — APPROVED** [review]

**Comment by jesuino** [comment]
There's an ElasticSearch provider for Dashbuilder and I see remaining classes in `appformer/dashbuilder`. I never saw anyone using Elastic Search data set provider in Dashbuilder, so it may not be an issue

**Pull request merged**